tests/v4/functional/test_redact_folder.py

```diff
@@ -64,7 +64,7 @@ def _replace_file_ext(file_path: Union[str, Path], new_ext: str = ".json") -> st
         "output_type,service,file_extension",
         [
             [OutputType.labels, ServiceType.redact_area, ".json"],
-            [OutputType.overlays, ServiceType.dnat, ".apng"],
+            [OutputType.overlays, ServiceType.dnat, ".png"],
```

Comment:
why just `png`?
If the output type is `overlays` it should be `apng`, right?

Reply:
Only for videos. I was actually debugging this for 30 min 🙈😅. apng stands for animated PNG. So for images this doesn't really make sense for images

Reply:
yes, found it in the `delivery`